src/main/java/uk/gov/justice/hmpps/prison/web/config/ResourceServerConfiguration.java

```diff
@@ -4,37 +4,42 @@
 import org.springframework.context.annotation.Configuration;
 import org.springframework.scheduling.annotation.EnableAsync;
 import org.springframework.scheduling.annotation.EnableScheduling;
+import org.springframework.security.config.annotation.method.configuration.EnableMethodSecurity;
 import org.springframework.security.config.annotation.web.builders.HttpSecurity;
-import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter;
+import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
 import org.springframework.security.config.http.SessionCreationPolicy;
+import org.springframework.security.web.SecurityFilterChain;
 import uk.gov.justice.hmpps.prison.security.EntryPointUnauthorizedHandler;
 
 @Configuration
 @EnableScheduling
 @EnableAsync(proxyTargetClass = true)
-public class ResourceServerConfiguration extends WebSecurityConfigurerAdapter {
+@EnableMethodSecurity(prePostEnabled = true, proxyTargetClass = true)
```

Comment:
this was moved from its own class - that's how the rest of our projects do it anyway